docs/create-support-bundle.md

```diff
@@ -0,0 +1,16 @@
+# Create Support Bundle Scripts
+
+There are three support bundle scripts that are used to collect information about the system and the collector. TThey must be run on the machine hosting the Collector. The scripts are located in the `scripts/support` directory. They produce output in the directory they are run from. All of the support scripts collect the following information:
```

Comment:
Can we add a link directory to `scripts/support`. You can do this by `[scripts/support](../scripts/support)`